dynawo/sources/Modeler/DataInterface/DYNVscConverterInterface.h

```diff
@@ -50,6 +50,12 @@ class VscConverterInterface : public ConverterInterface {
    * @return The voltage set-point in kV
    */
   virtual double getVoltageSetpoint() const = 0;
+
+  /**
+   * @brief Getter for the maximum reactive power of the converter
+   * @return The maximum reactive power of the converter in Mvar (following iidm convention)
+   */
+  virtual double getQMax() = 0;
```

Comment:
not const?

Reply:
I took the DYNGeneratoInterface as an example and it doesn't have any const

Reply:
I did the same as for the generator which doesn't have a const.
I tried with a const and got an error.